select_statements.sql

```diff
@@ -31,89 +31,60 @@ UPDATE vehicle
 
 -- Select all vacant cars for all stations
 SELECT v.registrationNumber as Registreringsnummer, vc.name as Fordonskategori, v.stationName as Station
-FROM vehicle as v
+FROM green_rental.vehicle as v
 LEFT JOIN booking_details as bd ON v.registrationNumber = bd.registrationNumber
 LEFT JOIN booking as b ON bd.bookingNumber = b.bookingNumber
 INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId
-WHERE b.endDate < "2024-04-13" OR b.endDate IS NULL
-ORDER BY Station, Fordonskategori;
+WHERE b.endDate < "2024-04-13" OR b.endDate IS NULL;
 
 -- Select all vacant cars for "my" station of a certain category
-SELECT v.registrationNumber, v.stationName FROM vehicle as v
+SELECT v.registrationNumber, v.stationName FROM green_rental.vehicle as v
 LEFT JOIN booking_details as bd ON v.registrationNumber = bd.registrationNumber
 LEFT JOIN booking as b ON bd.bookingNumber = b.bookingNumber
 INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId
 WHERE v.stationName = "Uppsala station" AND vc.name = "Stadsbil" AND (b.endDate < "2024-04-12" OR b.endDate IS NULL);
 
--- Select number of cars per station
-SELECT v.stationName as Station, count(*) as "Antal bilar" 
-FROM vehicle as v GROUP BY v.stationName;
-
--- Select all stations with vacant cars
-SELECT v.stationName as Station, count(*) as "Antal lediga bilar" 
-FROM vehicle as v 
-INNER JOIN booking as b ON v.stationName = b.stationName 
-WHERE v.registrationNumber IN 
-	(  
-		SELECT v.registrationNumber 
-        FROM vehicle as v  
-        LEFT JOIN booking_details as bd ON v.registrationNumber = bd.registrationNumber  
-        LEFT JOIN booking as b ON bd.bookingNumber = b.bookingNumber  
-        INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId  
-        WHERE b.endDate < "2024-04-12" OR b.endDate IS NULL 
-	) 
-GROUP BY v.stationName;
-
-
-
-
--- Underhållspersonal
 -- Sök fram alla bilar i behov av kontroll
-SELECT vehicle.registrationNumber FROM vehicle
+SELECT booking_details.bookingNumber as bokningsnummer, vehicle.registrationNumber as registreringsnummer, 
+booking.endDate as bokning_slutdatum, control.controlDate as senaste_kontrolldatum, 
+vehicle.stationName as station,control.staffID as senaste_kontroll_utförd_av_personalID FROM vehicle
 INNER JOIN booking_details ON vehicle.registrationNumber = booking_details.registrationNumber
 LEFT JOIN booking ON booking_details.bookingNumber = booking.bookingNumber
 LEFT JOIN control ON vehicle.registrationNumber=control.registrationNumber
-WHERE control.datum < "2024-04-12" and booking.endDate = "2024-04-12";
+WHERE booking.endDate = "2024-02-18" and (control.controlDate < "2024-02-18" or control.controlDate = null)
+ORDER BY bokning_slutdatum;
 
--- Sök fram alla bilar i behov av kontroll inom 3/6/12 månader
+-- Sök fram alla bilar i behov av stor kontroll
 SELECT vehicle.registrationNumber as Registreringsnummer
-FROM vehicle
+FROM green_rental.vehicle
```

Comment:
I removed `green_rental.` from all statements to make them easier to read in the report.